inference-engine/src/vpu/graph_transformer/src/middleend/allocator/allocator.cpp

```diff
@@ -172,6 +178,7 @@ bool Allocator::allocateData(const Data& data) {
                 finalByteSize = data->totalByteSize();
             } else {
                 finalByteSize = data->totalByteSize() * _modelBatchSize;
+                data->attrs().set("batch", _modelBatchSize);
```

Comment:
Nitpick: This section could be similar to one about inputs:

```suggestion
        auto finalByteSize = data->totalByteSize();
        if (data->attrs().getOrDefault<bool>("unbatched", false)) {
            finalByteSize *= _modelBatchSize;
            data->attrs().set("batch", _modelBatchSize);
        }
```